### PR TITLE
DS-6811: Update pds_storagesize_increase_test.go

### PR DESCRIPTION
**Pull request by jkagliwal-px**
100 iteration test for storage size increase. 

Steps performed by the testcase - 
1. Deploy  Data Service
2. Start Workload on  the DS deployed
3. Start re-sizing volume 1G at a time for custom number of iterations
4. Verify Volume size at every iteration

Jenkins Logs - https://jenkins.pwx.dev.purestorage.com/job/PDS/job/pds-branch-build-test/637/consoleFull

### CODE REVIEW

**Comment by Madanagopal19** [comment]
Add the steps it performs in the description

**Comment by Madanagopal19** [comment]
Add the steps it performs in the description

File: tests/pds/pds_storagesize_increase_test.go [hunk 1]
```diff
@@ -486,17 +476,16 @@ var _ = Describe("{PerformStorageResizeBy1Gb100TimesAllDs}", func() {
...
 				stepLog = "Check PVC for full condition based upto 90% full"
 				storageSizeCounter := 0
-				for i := 2; i <= 10; i++ {
-					CleanMapEntries(pdsdeploymentsmd5Hash2)
+				for i := 2; i <= 100; i++ {
+					tempstIds, tempresIds = nil, nil
+					dataserviceID, _ := dsTest.GetDataServiceID(ds.Name)
 					log.InfoD("The test is executing for the [%v] iteration", i)
 					storageSizeCounter = i
 					storageSize := fmt.Sprint(storageSizeCounter, "G")
```
Comment (Madanagopal19) [inline tests/pds/pds_storagesize_increase_test.go:491]: can we also add a logger for the current storage size ?

**Review by Madanagopal19 — CHANGES_REQUESTED** [review]

File: tests/pds/pds_storagesize_increase_test.go [hunk 2]
```diff
@@ -486,17 +476,16 @@ var _ = Describe("{PerformStorageResizeBy1Gb100TimesAllDs}", func() {
...
 				resIds = append(resIds, resConfigModel.GetId())
-				dataserviceID, _ := dsTest.GetDataServiceID(ds.Name)
+				deploymentsToBeCleaned = append(deploymentsToBeCleaned, deployment)
+				wlDeploymentsToBeCleaned = append(wlDeploymentsToBeCleaned, wlDep)
+				log.InfoD("workload deployment is- %v", wlDep.Name)
 				stepLog = "Check PVC for full condition based upto 90% full"
 				storageSizeCounter := 0
-				for i := 2; i <= 10; i++ {
-					CleanMapEntries(pdsdeploymentsmd5Hash2)
+				for i := 2; i <= 100; i++ {
```
[the remark this answers is not in the document]
Comment (jkagliwal-px) [reply tests/pds/pds_storagesize_increase_test.go:486]: Discussed offline

File: tests/pds/pds_storagesize_increase_test.go [hunk 3]
```diff
@@ -551,39 +538,28 @@ var _ = Describe("{PerformStorageResizeBy1Gb100TimesAllDs}", func() {
...
-							}
-						})
-						Step("Delete Deployments", func() {
-							CleanupDeployments(deploymentsToBeCleaned)
-							err := controlPlane.CleanupCustomTemplates(stIds, resIds)
-							log.FailOnError(err, "Failed to delete custom templates")
-						})
-
+						//ToDo: Re-run  Workload to check if I/O is running post update
+						//ToDo: Perform backup restores on updated dep
```
[the remark this answers is not in the document]
Comment (jkagliwal-px) [reply tests/pds/pds_storagesize_increase_test.go:542]: ok 
Ticket - https://portworx.atlassian.net/browse/DS-7018

**Review by Madanagopal19 — APPROVED** [review]
LGTM

**Review by aravindranath14 — APPROVED** [review]
Approved

**Pull request merged**